Peer/ClientThread.java

```diff
@@ -55,6 +55,17 @@ public void run()
             System.exit(1);
         }
 
+        //Ensuring that the target IP was received
+        if(targetIP.contains("NOTFOUND"))
+        {
+            System.err.println("The target IP was not found.\nExiting Program.");
+            System.exit(1);
```

Comment:
@trobe137 System.exit() will kill all threads. Replace this with something more appropriate, so that the receiving thread isn't killed?